quickwit-index-config/resources/tests/index_object/hdfs-logs.toml

```diff
@@ -0,0 +1,39 @@
+index_id = "hdfs-logs"
```

Comment:
Correctness aside, at the end of the day, TOML is my least favorite format to work with.

Reply:
haha really?
Should we switch to yaml then?